room.js

```diff
@@ -1,703 +0,0 @@
-let answer
-let allPossibleDirections = ['north', 'south', 'east', 'west']
-let keypad_answer
-
-
-
-/// MOST IMPORTANT OBJECTS AND FUNCTIONS /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-const readline = require('readline');
-const { userInfo } = require('os');
-const readlineInterface = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout
-})
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-      readlineInterface.question(questionText, resolve);
-    });
-  };
-
-function examine(object){
-    if(object.current_room === player.current_room && object.current_facing === player.current_facing){
-        console.log('\n \x1b[36m%s\x1b[0m', `\n${object.desc}`)
-    }
-}
-
-
-let player = {
-    inv: [],
-    health: 100,
-    status: 'fine',
-    isTrapped: true,
-    current_room: 'room1',
-    current_facing: 'north',
-
-    check_inventory(){
-        if(player.inv.length !== 0){
-            console.log('\n \x1b[36m%s\x1b[0m', `\n Current inventory: ${player.inv}`)
-        } else {
-            console.log('\n \x1b[36m%s\x1b[0m', `\n You don't have anything!`)
-        }
-        
-    },
-    escape(){
-        console.log("You have escaped")
-        process.exit()
-    },
-
-    damage(damage_num){
-        if((this.health - damage_num)<= 0){
-            this.health -= damage_num
-            this.game_over()
-        } else {
-            this.health -= damage_num
-        }
-        
-    },
-    status_effect(){
-        switch(this.status){
-            case 'hungry':
-                this.damage(5)
-                console.log('\n \x1b[33m%s\x1b[0m', `Your stomach growls in hunger, you need food fast! You have taken 5 damage and have ${player.health} health left!`)
-                break
-
-            case 'fine' && this.health < 100:
-                this.health += 5
-                console.log('\n \x1b[33m%s\x1b[0m', `Due to feeling fine, you are slowly healing! You've gained 5 health and are now at ${this.health} health!`)
-                break
-
-            case 'fine' && this.health === 100:
-                break
-
-            default:
-                console.log("Error, unusual status detected!")
-                break
-        }
-    },
```

Comment:
As of right now, as soon as you exit the room, your status defaults to the "Error,..." I would be curious to see how this and the damage functions work out